test/db/cmd/cmd_hash

```diff
@@ -40,4 +40,28 @@ CMDS=<<EOF
 EOF
 EXPECT=<<EOF
 EOF
+RUN
+
+NAME=#!c
+FILE=-
+CMDS=<<EOF
+"?e #include <stdio.h>\nint entry(){printf(\"Hello\\n\");}" > .tmp-hashc.c
+#!c .tmp-hashc.c
+rm .tmp-hashc.c
+EOF
+EXPECT=<<EOF
+Hello
+EOF
+RUN
+
+NAME=#! with ext != name
+FILE=-
+CMDS=<<EOF
+"?e #include <stdio.h>\nint main(){printf(\"Hello\\n\");}" > .tmp-hashextname.c
```

Comment:
what about adding those .c files instead of creating them at test time?

Reply:
as you prefer actually. i don't have strong opinions, this way was just faster for me because i don't really care about the content of the file, i just wanted to make sure things compiled properly.